modules/S15_radare_decompile_checks.sh

```diff
@@ -189,6 +192,13 @@ radare_decomp_print_top10_statistics() {
           fi
           if [[ -f "$LOG_PATH_MODULE""/vul_func_""$F_COUNTER""_""$FUNCTION"-"$SEARCH_TERM"".txt" ]]; then
             write_link "$LOG_PATH_MODULE""/vul_func_""$F_COUNTER""_""$FUNCTION"-"$SEARCH_TERM"".txt"
+            if [[ $GPT_OPTION -gt 0 ]]; then
+              GPT_PRIO=2
+              GPT_ANCHOR="$(openssl rand -hex 8)"
```

Comment:
could we just use "${RANDOM}"?

Reply:
I could change it to `"q02-""${RANDOM}"` or something but I'm not sure this would provide the same uniqueness